docs/log_collection.md

```diff
@@ -0,0 +1,181 @@
+# IOS Log Collection
+
+Send logs to Datadog from your IOS applications with [Datadog's `dd-sdk-ios` client-side logging library][1] and leverage the following features:
+
+* Log to Datadog in JSON format natively.
+* Add `context` and extra custom attributes to each log sent.
+* Record real client IP addresses and User-Agents.
+* Optimized network usage with automatic bulk posts.
+
+**Note**: The `dd-sdk-ios` library supports all IOS versions from 11.
+
+## Setup
+
+1. Declare the library as a dependency depending on your package manager:
+
+    {{< tabs >}}
+    {{% tab "CocoaPods" %}}
+
+You can use [CocoaPods][6] to install `dd-sdk-ios`
+```
+pod 'Datadog'
+```
+
+    {{% /tab %}}
+    {{% tab "Swift Package Manager (SPM)" %}}
+
+To integrate using Apple's Swift package manager, add the following as a dependency to your Package.swift:
+```swift
+.package(url: "https://github.com/Datadog/dd-sdk-ios.git", .upToNextMajor(from: "0.0.0"))
+```
+
+    {{% /tab %}}
+    {{% tab "Carthage" %}}
+Not yet supported
```

Comment:
If there are no very immediate plans to support carthage, let's just remove this tab